heuristic/delta_calculations.py

```diff
@@ -417,4 +417,127 @@ def regret_objective_function(state, employee, off_shifts, saturdays, days, L_C_
             - 10 * state.hard_vars["delta_positive_contracted_hours"].get(employee, 0)
 
             - 100 * competency_score
+    )
+
+"""
+        What is important to calculate?
+            1. deviation_from_demand vil ikke ha noe å si. Derfor unyttig å kalkulere
+           
+            8. Shift to demand burde ikke trenges ettersom vi sjekker om en ansatt jobber den dagen. 
+            9. Cover multiple demand periods er likt den over. Burde ikke trenge å sjekkes. 
+            10. More than one shift per day burde ikke trenge det heller ettersom vi sjekker et annet sted.
+            11. above/below ikke nødvendig det heller. 
+            
+            13. Dette er opp mot objektivverdi og trenger da sannsynligvis ikke å regne egen objektivverdi. Kan bare se på hver ansatt. 
+        """
+"""
+            2. weekly rest. Når vi setter inn et skift så bør denne kalkuleres. Bør derimot strengt tatt ikke sette verdien, men heller regne ut høyeste mulige verdi.
```

Comment:
I think it could be beneficial to store the value. This will give us the opportunity to check if a shift allocation would reduce or break weekly rest. However, it is not urgent.

Reply:
I don't think I follow. The value is only the longest possible continuous rest period for an employee when being assign a shift with his current assignments. The actual value is stored later in the soft_vars dict

Reply:
Which variable? 

![image](https://user-images.githubusercontent.com/1371159/82031401-2221ec80-969a-11ea-81d6-786233fed31c.png)

Reply:
Was wrong. The weekly rest is stored in the w-dict, not the soft_vars dict.

Reply:
Makes sense! This solves some problems for me! Thanks